File: libs/common/models/vw-pmp-psispecialization-count-by-category.ts



export class VwPmpPsispecializationCountByCategory {
    public cipSubSeriesCode: string = '';
    public cipSubSeries: string = '';
    public programCount: number;
    public specailizationCount: number;
    public sessionCount: number;
    public providerCount: number;

    constructor(init? : Partial<VwPmpPsispecializationCountByCategory>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/EntityFramework/VwPmpPsispecializationCountByCategory.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class VwPmpPsispecializationCountByCategory
    {
        public string CipSubSeriesCode { get; set; }
        public string CipSubSeries { get; set; }
        public int? ProgramCount { get; set; }
        public int? SpecailizationCount { get; set; }
        public int? SessionCount { get; set; }
        public int? ProviderCount { get; set; }
    }
}
